ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -876,6 +876,21 @@ literals:
 - fire_alarm:
   - ACTIVE
   - INACTIVE
+- primary_fire_alarm: # better than using fire_alarm_1 (which will lead to fire_alarm_1_23)
```

Comment:
explain what is meant by a primary fire alarm. how do you have more than one fire alarm? either there is a fire or there is not.

Reply:
Hi sir, I'm thinking in a firestation, there are generally several types of alarms to indicate how big the fire is or how large an area is affected (in the news e.g., they call it 1st alarm response or 2nd alarm response, etc).  I assume this is the case for these devices (Novec123 fire suppression, and aspirating type smoke detection) as they do indeed have a 1st alarm & 2nd alarm in their points list. 
Then I see we have multiple aspirating types connected in a DDC so their point names (fire_alarm_1 for 1st alarm, fire_alarm_2 for 2nd alarm) would be enumerated (e.g., fire_alarm_1_23, fire_alarm_2_23 for aspirating type device 23). And that looks confusing. So instead of that (fire_alarm_1) i used primary_alarm and secondary_alarm as I can't find first and second in description subfields.

Reply:
from actual device:

![image](https://user-images.githubusercontent.com/21256796/228437608-49dba9a4-a407-4897-ae01-4d3da6a7ea9f.png)

For the above points list:
first/second alarm is by area. In the hardware there are 2 sensors (on opposite sides) that when a sensor gets triggered, the first alarm goes on. When the other sensor gets triggered, second alarm goes on. It is as if the area is being monitored, and a first alarm means a small area is affected. a second alarm means a larger area is affected. Supervisor/Trouble alarm will be renamed probably to failed_panel_alarm

--------------------------------------
For aspirating types:
Different from the fire suppression devices above, it has 3 alarms for action and 1 alarm for internal monitoring (like above: failed_panel_alarm)

![image](https://user-images.githubusercontent.com/21256796/228437852-0e602874-24a5-419b-b29f-8dc4b9e31613.png)

These alarms (except supervisor/trouble) are programmable and are based on accumulation of smoke detected. Since these are programmable thresholds, thresholds are usually set low for very early detection, and progressively for the other alarms. We could say these alarms are variations of smoke intensity. I'll just use low_smoke_alarm (very early detection), medium_smoke_alarm (early detection), fire alarm (as is, instead of high_smoke_alarm) and failed_panel_alarm (which means an error in device itself)